src/momento/internal/aio/_scs_data_client.py

```diff
@@ -189,6 +190,9 @@ async def increment(
             )
             self._log_received_response("Increment", {"key": str(key), "amount": str(amount)})
             return CacheIncrement.Success(response.value)
+        except grpc.RpcError as rpc_error:
+            self._log_request_error("increment", rpc_error)
+            return CacheIncrement.Error(convert_error(rpc_error, Service.CACHE, rpc_error.trailing_metadata()))
```

Comment:
Would we just modify convert_error to read the metadata from `trailing_metadata` when the type is `grpc.RpcError`? Note that function does much processing and normalization like that already.